scanner/src/funTest/kotlin/scanners/ScannerIntegrationFunTest.kt

```diff
@@ -100,6 +101,18 @@ class ScannerIntegrationFunTest : WordSpec({
                 patchExpectedResult(expectedResult)
         }
     }
+
+    "Scanning a project with the same provenance as packages" should {
+        val analyzerResult = createAnalyzerResultWithProject(project0, pkg0)
+        val ortResult = createScanner().scan(analyzerResult, skipExcluded = false, emptyMap())
```

Comment:
Please move lines 106+107 below line 109.

Reply:
Done in ed52821428f2b979de59f9b12dca9fe53792cdf0